packages/headless/src/features/search-parameters/search-parameter-serializer.ts

```diff
@@ -39,7 +39,13 @@ function serializePair(pair: [string, unknown]) {
     return '';
   }
 
-  if (key === 'f' || key === 'cf' || key === 'sf' || key === 'af') {
+  if (
```

Comment:
This is reaching the limit of readability for an inline boolean check, IMO.

Suggestion: could extract an utility function for `isSpecificFacetKey` (specific meaning, the Search API type for "normal" or "standard" facet)

Reply:
I was suggesting having something like this:
```typescript
if(isSpecificFacetKey(key)) {
   return isFacetObject(val) ? serializeFacets(key, val) : '';
}
```